esphome/components/ina226/sensor.py

```diff
@@ -110,7 +112,8 @@ async def to_code(config):
 
     cg.add(var.set_shunt_resistance_ohm(config[CONF_SHUNT_RESISTANCE]))
     cg.add(var.set_max_current_a(config[CONF_MAX_CURRENT]))
-    cg.add(var.set_adc_time(config[CONF_ADC_TIME]))
+    cg.add(var.set_adc_time_voltage(config[CONF_ADC_TIME_VOLTAGE]))
+    cg.add(var.set_adc_time_current(config[CONF_ADC_TIME_CURRENT]))
```

Comment:
This is paired with the above schema change
```suggestion
	adc_time_config = config[CONF_ADC_TIME]
	if isinstance(adc_time_config, dict):
    	cg.add(var.set_adc_time_voltage(adc_time_config[CONF_VOLTAGE]))
    	cg.add(var.set_adc_time_current(adc_time_config[CONF_CURRENT]))
    else:
    	cg.add(var.set_adc_time_voltage(adc_time_config))
    	cg.add(var.set_adc_time_current(adc_time_config))
```